README.md

```diff
@@ -1,19 +1,34 @@
-# vRealize Automation Plug-in Starter Content Pack
+# vRealize Automation Plug-in Starter Content Pack for vRA 7.3 and Puppet plug-in 3.0
 
 ## Introduction
 
-This repository provides starter Puppet content for the [Puppet plug-in for vRealize Automation](https://solutionexchange.vmware.com/admin/products/133777) (vRO and vRA). It shows you how Puppet code is organized to make self-provisioning easy and can create 5 different machine types out of the box:
+This repository provides "batteries-included" starter Puppet content for the [Puppet plug-in for vRealize Automation 3.0](https://solutionexchange.vmware.com/store/products/puppet-plugin-for-vrealize-automation). It shows you how Puppet code is organized to make self-service provisioning easy and can create 5 different machine types out of the box:
 - Linux base server
 - Linux MySQL server
 - Linux web server
 - Windows base server
 - Windows web server
 
-It also provides two example [vRA Blueprints](https://github.com/puppetlabs/puppet-vro-starter_content/tree/production/blueprints) designed to work with the Puppet content.
-- PELinuxWebserverExample-composite-blueprint.zip
-- PEWindowsWebserverExample-composite-blueprint.zip
+With vRA 7.3 and the Puppet plug-in for vRealize Automation 3.0, building Puppet Enterprise blueprints is easier than ever with PE components now built directly into the vRA Enterprise GUI. While creating blueprints, you simply drag and drop the Puppet component onto the blueprint and fill in a couple of text boxes. No need to jump back and forth from vRA to vRO to get set up. Even better, Puppet role classes are now read dynamically from your Puppet Server, so you need only pick from the list instead of typing classes out manually.
 
-Please see the [complete documentation](https://docs.puppet.com/pe/latest/vro_intro.html) for more information on how to install the Puppet plug-in and use the content provided in this repository.
+## Getting started
+
+Here's your high-level plan to get started:
+- Create a Puppet Enterprise master and follow the instructions at the bottom of this page to install the starter content.
+- Remove any previous Puppet plug-ins and install the [Puppet plug-in for vRealize Automation 3.0](https://solutionexchange.vmware.com/store/products/puppet-plugin-for-vrealize-automation) into the embedded vRO that comes in your vRA appliance (external vRO servers are not supported, use the legacy workflow if you need this).
```

Comment:
I need to word this better. I believe that only the interval vRO is supported with the vRA/Puppet GUI integration. External vRO's will still work with Event Broker.